### PR TITLE
Add travis

### PR DESCRIPTION
**Pull request by asvetlov**
@Axik please review

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  - rm -f $HOME/.cache/pip/log/debug.log
+
+after_success:
+  - coveralls
+
+deploy:
+  provider: pypi
+  user: andrew.svetlov
+  password:
+    secure: "RDqrg/WzHF84GWejs0l7yo0C+vGt0LWkqxW1YgGayQOOBkflii3AzdBimVDoWmYGp7GR7bbWDarj5hmEYfs9CBRaH34OWPr8D811bUDyChI9HGJoeeaY0fhOO9sawfpP1U/ovwOz/gqCdq+X9m5Fk2H91RBy8JpH/hA6q/XtqJAQ1WXY0AYPsAIFhva19VPr5Tgg5Vg7H0MzEA65Bnk3t6MPYVR+dg3rRS33e+BtqFbQuetPOubfdceVZRjOLgDHh9ztEhES7XXH5ORpD1zBgcZTlTVyZs03XthgCv82bGjtBgvVK5vqJ1cGmYT2hySczBZRI9gCjQJ3R8ug0sjXpqoTw8dfkzfZAzbn7v2jdJUV7pOLdce8YJi4gbyJk5Vipsdux6D8zCeSmPGzZ7+E0DoLsgXBCpqlB1JqCda1KT+3BrB/xJLIj10lWwqbnN+PwyVKKjp4R0mERKctKw4uGC5aVGg3jPXIs+B4bUvTieBUe4ZZBWBRRXbzE9+ewuobhxSK4AfBybyVRmOJvY75+60vsbSsta7/xDrCf3glR2dSfk1YfMUzoDq6De8Iw8f66bSnvdqGXcgWiT4izvk/hPhVIlRUzyeosX4EsNlKubJHkmHIBqeoYho+mGpPycynr1fAqSHW/xUUeoV+MjfMPdfRIHWVsclNR4aNSlgQQi0="
```
Comment (alexeyraspopov) [inline .travis.yml]: Is it ok?

**Comment by asvetlov** [comment]
@Axik Finally fixed hungs

File: .travis.yml (hunk 1 above)
Comment (asvetlov) [inline .travis.yml]: Yes

**Comment by Axik** [comment]
@asvetlov 
LGTM
:ship::it:

**Pull request merged**